ziti/internal/edge_impl/conn.go

```diff
@@ -214,18 +209,18 @@ func (conn *edgeConn) Listen(session *edge.Session, serviceName string, options
 	conn.TraceMsg("listen", bindRequest)
 	replyMsg, err := conn.SendAndWaitWithTimeout(bindRequest, 5*time.Second)
 	if err != nil {
-		logger.Error(err)
+		logger.WithError(err).Error("failed to bind")
```

Comment:
you have the serviceName in this function - think these errors should all have that service name in them just in case it's useful down the road?

Reply:
It's already there... the logger is defined above as

```
	logger := pfxlog.Logger().
		WithField("connId", conn.Id()).
		WithField("service", serviceName).
		WithField("session", session.Token)
```